addons/editor/initKeybinds.sqf

```diff
@@ -45,6 +45,14 @@
     };
 }, {}, [DIK_V, [true, true, false]]] call CBA_fnc_addKeybind; // Default: CTRL + SHIFT + V
 
+[ELSTRING(common,Category), QGVAR(focusSearchBar), [LSTRING(FocusSearchBar), LSTRING(FocusSearchBar_Description)], {
+    if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false)}) then {
+        private _searchIDC = [IDC_RSCDISPLAYCURATOR_CREATE_SEARCH, IDC_SEARCH_CUSTOM] select GVAR(disableLiveSearch);
+        private _ctrlSearch = findDisplay IDD_RSCDISPLAYCURATOR displayCtrl _searchIDC;
+        ctrlSetFocus _ctrlSearch;
+    };
+}, {}, [DIK_F, [false, true, true]]] call CBA_fnc_addKeybind; // Default: CTRL + ALT + F
```

Comment:
<kbd>Ctrl</kbd> + <kbd>F</kbd> wouldn't work?

Reply:
It's the default for the "Lock camera on selected entity" keybind.

Reply:
I see :)